### ISSUE TITLE
Failed to compile due to file React95 dependency 

### ISSUE THREAD

**Issue by acupoftee** [React95/React95#331]
I began a new project with React95 using React v18.0.0. I was trying to compile a simple App component to test my setup and received the following error:

```
Failed to compile.

./node_modules/@xstyled/styled-components/dist/index.mjs
Can't import the named export 'ThemeContext' from non EcmaScript module (only default export is available)
```

Here's the code that I've tried to compile below. I've only modified `App.js` after creating the project with `create-react-app`.

```jsx
import React from 'react';
import './App.css';
import { Button, ThemeProvider } from "@react95/core";


function App() {
  return (
      <ThemeProvider>
        <Button>Test</Button>
      </ThemeProvider>
  )
}

export default App;
```

Thank you in advance for your help!

**Comment by ggdaltoso** [comment]
Hi, @acupoftee!

Thanks for bringing me this issue!

Which version of @react95/core are you using?

**Comment by ggdaltoso** [comment]
I reproduce your issue and I'm working on it!

**Comment by ggdaltoso** [comment]
@acupoftee This is has been fixed by #334 and now you're ready to go.

Please, feel free to contribute even more! You can always open new issues or create a pull request!

🎉
